corsy.py

```diff
@@ -54,7 +56,10 @@
         'Connection': 'close',
     }
 
-urls = create_url_list(target, inp_file)
+if inp_file:
+    urls = create_url_list(target, inp_file)
+elif log_file:
+    urls = url_from_logs(log_file)
```

Comment:
`create_url_list` must be executed even if there's no input file added by the user. Hence, the code here should be:
```python
if log_file:
    urls = url_from_logs(log_file)
else:
    urls = create_url_list(target, inp_file)
```